### DISCUSSION TITLE
What does the star/number mean?

### DISCUSSION THREAD

**Discussion by Integralist** [starship/starship#3756] [Q&A]
<img width="75" alt="Screenshot 2022-03-16 at 11 15 59" src="https://user-images.githubusercontent.com/180050/158578247-18401737-8a9c-4cae-939f-3a056f0640cb.png">

I think the blue star indicates there is a background job (which there is in my case) but the number `2` I don't know what that means because if I run `jobs` I just see one job:

```
[1]+  Stopped                 vim
```

Thanks.

**Comment by davidkna** [comment]
It is possible that a shell script is spawning jobs during prompt rendering that aren't visible after the prompt has been printed.

**Reply by Integralist** [reply]
So I guess my understanding is correct then, in that the blue star does indeed mean there's (supposedly) a background job and the number next to it represents how many jobs.

You're likely correct and it's just something in my configuration that's causing these ghost jobs to be displayed. 

Unfortunately it's actually quite tricky to nail down because of the various scripts that get sourced via various external tools (e.g. https://fig.io or https://github.com/junegunn/fzf etc).
